src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-overview-dashboard/rgw-overview-dashboard.component.html

```diff
@@ -92,20 +92,185 @@
       <cd-card cardTitle="Used Capacity"
                i18n-title
                class="col-sm-2 d-flex w-100 h-50 pb-3"
-               aria-label="Details card">
+               aria-label="Used Capacity">
         <span  class="ms-4 me-4 text-center">
           <h1>{{ totalPoolUsedBytes | dimlessBinary}}</h1>
         </span>
       </cd-card>
       <cd-card cardTitle="Avg Object Size"
                i18n-title
                class="col-sm-2 d-flex w-100 h-50 pt-3"
-               aria-label="Details card">
+               aria-label="Avg Object Size">
         <span class="ms-4 me-4 text-center">
           <h1>{{ averageObjectSize | dimlessBinary}}</h1>
         </span>
       </cd-card>
     </div>
   </div>
+  <div class="row pt-4">
+    <cd-card cardTitle="Multisite Sync Status"
+             i18n-title>
+      <div *ngIf="!showMultisiteCard"
+           class="text-center">
+           No Data!
+      </div>
+      <div class="row"
+           *ngIf="showMultisiteCard">
+        <div class="row pt-4">
+          <cd-card cardTitle="Primary Source Zone"
+                   class="col-lg-3 d-flex justify-content-center align-primary-zone">
+            <span *ngIf="loading"
+                   class="d-flex justify-content-center">
+            <i [ngClass]="[icons.large3x, icons.spinner, icons.spin]"></i></span>
+            <span *ngIf="!loading"
+                  class="d-flex justify-content-center">
```

Comment:
I have never seen this `loading spinner` in my view. and I am guessing its because of the `showMultisiteCard` condition which prevents the card from being shown in the first place.

Reply:
Fixed